src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -201,15 +201,18 @@ double RNN<OutputLayerType, InitializationRuleType, CustomLayers...>::Evaluate(
   ResetCells();
 
   double performance = 0;
+  size_t respSeq = 0;
```

Comment:
Do you mind to use a more descriptive name for the parameter e.g. ``responsesSeq``?